Assets/Scripts/PlayerControls.cs

```diff
@@ -4,16 +4,24 @@
 [RequireComponent( typeof( ActorPhysics ) )]
 public class PlayerControls : MonoBehaviour
 {
+	[Tooltip( "The distance forward from the camera's position to check for objects that can be interacted with." )]
+	[SerializeField] float _interactCheckDistance = 5.0f;
+	[SerializeField] float _interactCheckRadius = 0.2f;
+
 	PlayerActor _actor;
 	ActorPhysics _actorPhysics;
+	Cutting _cutting;
+	bool _holdButtonDown = false;
```

Comment:
Yeah. My logic is that the climbing and gliding buttons are held down during use, while the cutting and giving actions only require you to press the button once, so I've called the two actions "Hold" and "Use".